### PR TITLE
fix deprications brought about by ruby 2.7.0

### PR DESCRIPTION
**Pull request by andrewkress**
This should resolve issue #399 
to suppress the warnings until it gets merged put this in your Gemfile:
`gem 'json', github: 'andrewkress/json', branch: 'fix-ruby-deprications'`

### CODE REVIEW

File: json-java.gemspec [hunk 1]
```diff
@@ -10,7 +10,6 @@ spec = Gem::Specification.new do |s|
   s.email = "dev+ruby@mernen.com"
   s.homepage = "http://json-jruby.rubyforge.org/"
   s.platform = 'java'
-  s.rubyforge_project = "json-jruby"
```
Comment (dentarg) [inline json-java.gemspec:13]: This change as already been made: https://github.com/flori/json/commit/7e1e97dc11c5aaff76580310b7402ca13f5e62e2

**Comment by SamuSan** [comment]
Any chance there might some movement on this? We have warnings flooding our logs in multiple apps right now.

**Comment by marcandre** [comment]
This is fixed in the latest `json` 2.3.0

This PR can be closed.

**Comment by SamuSan** [comment]
<3

**Pull request closed without merging**